packages/calcite-components/src/components/navigation/navigation.e2e.ts

```diff
@@ -67,4 +69,42 @@ describe("calcite-navigation", () => {
     await hamburgerMenu.click();
     expect(eventSpy).toHaveReceivedEventTimes(3);
   });
+
+  describe("theme", () => {
+    const navigationHtml = html` <calcite-navigation navigation-action>
+      <calcite-navigation-logo heading="Walt's Chips"> </calcite-navigation-logo>
+    </calcite-navigation>`;
+
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-navigation-action-background-color": {
+          shadowSelector: `.${CSS.container} >>> .${CSS.containerContent} >>> calcite-action >>> button`,
```

Comment:
For mapped tokens, you should assert on the `action` token and avoid piercing through the shadow DOM (see [example](https://github.com/Esri/calcite-design-system/pull/9137/files#diff-4fa673c7e0163203ec11e23373522add5f8b274a3cef9bb65fc9e148a83f2962R278-R281)). 

Applies to all mapped tokens.

Reply:
Is it suggested to query the nested component rather than class and assert it tokens?

Example:
```
 "--calcite-navigation-user-avatar-text-color": {
          shadowSelector: `calcite-avatar`,
          targetProp: "--calcite-avatar-text-color",
        },
```
Vs        
        
```
 "--calcite-navigation-user-avatar-text-color": {
          shadowSelector: `.button`,
          targetProp: "--calcite-avatar-text-color",
        },
```

To support the second option the nested component should have the css class selector 
```
.button {

calcite-avatar {
//tokens
}
}